Tools-Override/tests.targets

```diff
@@ -28,13 +28,15 @@
 
   <PropertyGroup>
     <XunitRuntimeConfig>$(ToolsDir)\xunit.console.netcore.runtimeconfig.json</XunitRuntimeConfig>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT' and '$(RUNTIME_PATH)' != ''">%RUNTIME_PATH%\</TestRuntimeEnvVar>
```

Comment:
I think it might be easier to just set StartProgram directly to $(TestSharedFxDir)dotnet.exe